app/src/features/rules/screens/rulesList/components/RulesList/components/RulesTable/components/RulesColumn/RulesColumn.tsx

```diff
@@ -33,6 +34,10 @@ const RuleNameColumn: React.FC<{
           state={{ source: "my_rules" }}
           to={`${PATHS.RULE_EDITOR.EDIT_RULE.ABSOLUTE}/${record.id}`}
           onClick={() => {
+            if (record.isSample) {
+              trackSampleRuleEditorViewed(record.name);
+            }
+
```

Comment:
This should be a sample rule clicked event, instead of viewed. Viewed event can be called in the RuleEditor screen component